pkg/controller/stackconfigpolicy/controller_test.go

```diff
@@ -134,6 +168,27 @@ func TestReconcileStackConfigPolicy_Reconcile(t *testing.T) {
 	orphanSecretFixture.Name = "another-es-es-file-settings"
 	orphanSecretFixture.Labels["elasticsearch.k8s.elastic.co/cluster-name"] = "another-es"
 
+	orphanElasticsearchConfigSecretFixture := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      esv1.StackConfigElasticsearchConfigSecretName("another-es"),
+			Namespace: "ns",
+			Labels: map[string]string{
+				"elasticsearch.k8s.elastic.co/cluster-name":     "another-es",
+				commonlabels.StackConfigPolicyOnDeleteLabelName: commonlabels.OrphanObjectDeleteOnPolicyDelete,
+				reconciler.SoftOwnerNamespaceLabel:              policyFixture.Namespace,
+				reconciler.SoftOwnerNameLabel:                   policyFixture.Name,
+				reconciler.SoftOwnerKindLabel:                   policyv1alpha1.Kind,
+			},
+		},
+	}
+
+	orphanSecretMountsSecretFixture := getSecretMountSecret(t, esv1.ESNamer.Suffix("another-es", "test-secret-mount"), "ns")
+	orphanSecretMountsSecretFixture.Labels["elasticsearch.k8s.elastic.co/cluster-name"] = "another-es"
+	orphanSecretMountsSecretFixture.Labels[commonlabels.StackConfigPolicyOnDeleteLabelName] = commonlabels.OrphanObjectDeleteOnPolicyDelete
+	orphanSecretMountsSecretFixture.Labels[reconciler.SoftOwnerNamespaceLabel] = policyFixture.Namespace
+	orphanSecretMountsSecretFixture.Labels[reconciler.SoftOwnerNameLabel] = policyFixture.Name
+	orphanSecretMountsSecretFixture.Labels[reconciler.SoftOwnerKindLabel] = policyv1alpha1.Kind
```

Comment:
Would I not make sense to put all of this into the `getSecretMountSecret` helper to make the test setup less noisy here.